doc/zypper.8.txt

```diff
@@ -455,6 +455,19 @@ include::{incdir}/option_Solver_Flags_Common.txt[]
 		No automatic removal of unneeded dependencies.
 --
 
+*purge-kernels* ['options']::
+	Autoremoves installed kernels.
+	+
+	Automatically cleans up installed kernels according to the rules defined in /etc/zypp/zypp.conf:multiversion.kernels which can be given as <version>, latest(-N), running, oldest(+N).
++
+--
+	*--details*::
+		Show the detailed installation summary.
+
+	*-D*, *--dry-run*::
+		Test the removal of packages, do not actually remove anything. If used together with *--download-only* a meaningful file conflict check can be performed (see section *Package File Conflicts*).
+--
```

Comment:
Here and in the `remove` command we should IMO remove the `If used together with....` sentence. Remove has no download-only. It's somewhat confusing here.

Reply:
Done